pipelines/openjdk8_nightly_pipeline.groovy

```diff
@@ -1,13 +1,14 @@
 println "building ${JDK_VERSION}"
 
-def buildPlatforms = ['Mac', 'Windows', 'Linux', 'zLinux', 'ppc64le', 'AIX']
+def buildPlatforms = ['Mac', 'Windows', 'Linux', 'zLinux', 'ppc64le', 'AIX', 'aarch64']
```

Comment:
I'm not sure if we should add arm64 job to pipeline as it's still unstable? https://ci.adoptopenjdk.net/job/openjdk8_build_arm64_linux/

Reply:
in which case, I can remove this change.  I would keep the java8 release pipeline change as its already added there

Reply:
It should be stable now :)